### ISSUE TITLE
2D detection results on MPI_INF_3DHP dataset

### ISSUE THREAD

**Issue by sdxass** [Nicholasli1995/EvoSkeleton#8]
Hi, thank you for your great work!

Could you please provide the pre-processed 2D detections and 3D poses for MPI_INF_3DHP test set. As such, we can evaluate the model generalizability under cross-dataset setup.

Thank you!

**Comment by Nicholasli1995** [comment]
> Hi, thank you for your great work!
> 
> Could you please provide the pre-processed 2D detections and 3D poses for MPI_INF_3DHP test set. As such, we can evaluate the model generalizability under cross-dataset setup.
> 
> Thank you!

Hi, following previous works on 2D-to-3D networks (e.g., mixture density network), we directly use the provided key-points in MPI-INF-3DHP as inputs to evaluate the cross-dataset generalization performance of our 2D-to-3D model (stage 2) trained on Human 3.6M. We did not fine-tune the heatmap regression model on MPI-INF-3DHP. Thus you can directly use the official key-points in 3DHP for evaluation.

**Comment by sdxass** [comment]
Thanks for your quick reply!

**Comment by sdxass** [comment]
Thanks for your quick reply!
